streamer/src/quic.rs

```diff
@@ -700,8 +702,12 @@ mod test {
             let mut s1 = conn1.connection.open_uni().await.unwrap();
             let mut s2 = conn2.connection.open_uni().await.unwrap();
             s1.write_all(&[0u8]).await.unwrap();
+            s2.write_all(&[0u8]).await.unwrap();
             s1.finish().await.unwrap();
-            s2.write_all(&[0u8])
+            // Wait for a few milliseconds to allow connection close state to
+            // propagate from server to the client
+            sleep(Duration::from_millis(10)).await;
```

Comment:
This seems a little brittle. Could we block_on (with timeout) the connection close instead?